app/views/content/is-teaching-right-for-me/all-subjects.md

```diff
@@ -0,0 +1,299 @@
+
+You can <a href="https://www.thenational.academy/teachers/programmes/english-secondary-ks3/units">explore what teaching an English lesson could look like</a>.
+
+<a href="https://www.find-postgraduate-teacher-training.service.gov.uk/results?age_group=secondary&applications_open=true&can_sponsor_visa=false&has_vacancies=true&l=2&subjects%5B%5D=Q3&visa_status=false">Search for English teacher training courses</a>.
+
+## French
+
+<a href="https://getintoteaching.education.gov.uk/funding-and-support/scholarships-and-bursaries?funding_widget%5Bsubject%5D=geography&button=#funding-widget">Tax-free scholarships of £27,000 or bursaries of £25,000 are available for eligible trainee French teachers</a>. 
+
+As a French teacher you'll help pupils learn a new language and open them up to a different culture.
+
+<a href="https://www.find-postgraduate-teacher-training.service.gov.uk/results?age_group=secondary&applications_open=true&can_sponsor_visa=false&has_vacancies=true&l=2&qualification%5B%5D=qts&qualification%5B%5D=pgce_with_qts&qualification%5B%5D=pgce+pgde&send_courses=false&study_type%5B%5D=full_time&study_type%5B%5D=part_time&subjects%5B%5D=15&visa_status=false">Search for French teacher training courses</a>.
+
+## Geography 
+
+<a href="https://getintoteaching.education.gov.uk/funding-and-support/scholarships-and-bursaries?funding_widget%5Bsubject%5D=geography&button=#funding-widget">Tax-free bursaries of £25,000 are available for eligible trainee geography teachers</a>. 
+
+As a geography teacher you'll inspire curiosity in your pupils for the world and the people in it. While teaching them about the formation and use of landscapes and environments.
+
+ <a href="https://www.find-postgraduate-teacher-training.service.gov.uk/results?age_group=secondary&applications_open=true&can_sponsor_visa=false&has_vacancies=true&l=2&subjects%5B%5D=F8&visa_status=false">Search for Geography teacher training courses</a>.
+
+ ## German 
+ 
+ <a href="https://getintoteaching.education.gov.uk/funding-and-support/scholarships-and-bursaries?funding_widget%5Bsubject%5D=languages&button=#funding-widget">Tax-free scholarships of £27,000 or bursaries of £25,000 are available for all eligible trainee German teachers</a>.
+
+ As a German teacher you'll help pupils learn a new language and open them up to a different culture.
+
+<a href="https://www.find-postgraduate-teacher-training.service.gov.uk/results?age_group=secondary&applications_open=true&can_sponsor_visa=false&has_vacancies=true&l=2&qualification%5B%5D=qts&qualification%5B%5D=pgce_with_qts&qualification%5B%5D=pgce+pgde&send_courses=false&study_type%5B%5D=full_time&study_type%5B%5D=part_time&subjects%5B%5D=17&visa_status=false">Search for German teacher training courses</a>.
+
+## Health and social care 
+
+As a health and social care teacher you'll help pupils to understand aspects of personal development and teach them about major life changes and sources of support. 
```

Comment:
@caitlin-reidy can we reword to make it more conversational? I'm not sure what you mean by aspects of personal development? 

How do you teach someone about a major life change? Wouldn't it just happen? Do you mean how to cope with major life changes? And can we give an example? 

'sources of support' again can we make it more conversational e.g. where you can find support?